cli/slsa-verifier/main.go

```diff
@@ -35,6 +35,7 @@ For more information on SLSA, visit https://slsa.dev`,
 	}
 	c.AddCommand(version.Version())
 	c.AddCommand(verifyArtifactCmd())
+	c.AddCommand(verifyBCRModuleCmd())
```

Comment:
It looks like the workflow is just using `actions/attest-build-provenance` to create an attestation.

https://github.com/bazel-contrib/.github/blob/eeb03fb7518fb0036e4bb0713134c3b6531c68ba/.github/workflows/release_ruleset.yaml#L86-L91

What about this requires specific command for bcr modules? Is it just that we are trusting a specific builder ID? Could this be a more generic command? If we want to limit verifying provenance generated by `actions/attest-build-provenance` to specific trusted builder IDs, could we just have an option like `slsa-verifier verify-github-attestation --builder bazel-central-registry` or something along those lines?

Reply:
I can change this to `verify-github-attestation` , I'm not deep enough into slsa land to understand the consequences here. Is the `builderId` spoofable in any way?

Reply:
oh right, builderId nees to match the SAN in the certificate

Reply:
Yes. It's not spoofable because we check it against the certificate SAN (at least for sigstore which is all we support currently).

If we implemented private PKI in the future the consumer would presumably trust that the private key is only available to the build system that set the builder ID in that case.